src/components/Privacy/index.js

```diff
@@ -32,14 +42,22 @@ const createPrivacy = ({ config, consent, sendEdgeNetworkRequest }) => {
     createConsentRequest,
     sendEdgeNetworkRequest
   });
+  const storage = createNamespacedStorage(`consentHashes.${orgId}.`);
```

Comment:
Did you consider leading with the org ID? Leading with the org ID would probably be more consistent with [how we're leading with the the instance name when storing whether debugging is enabled](https://github.com/adobe/alloy/blob/2bee677f4b7296c936d64aeee2a419c786cd23f9/src/core/createLogController.js#L25-L27). In other words, in the case of storing whether debugging is enabled, the key is `com.adobe.alloy.instance.myInstanceName.debug` instead of something like `com.adobe.alloy.debug.instance.alloy`. So, for org-related items, `com.adobe.alloy.123F60F35E1597910A495EC2@AdobeOrg.consentHashes` might be more consistent than `com.adobe.alloy.consentHashes.123F60F35E1597910A495EC2@AdobeOrg`. Personally, I kind of prefer to have this grouped by the org.

Related: We are swapping the `@` for `_` when dealing with org names inside cookie names (see `sanitizeOrgIdForCookieName.js`), largely to avoid (undue) security notices (I believe). We might need to do the same for web storage keys for the same reason.